spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/source/TestSparkReaderDeletes.java

```diff
@@ -362,6 +388,8 @@ public void testEqualityDeleteWithDeletedColumn() throws IOException {
         rowSet(tableName, PROJECTION_SCHEMA.asStruct(), "id", "data", "_deleted");
 
     Assert.assertEquals("Table should contain expected row", expected, actual);
+    long expectedDeletes = 3L;
+    checkDeleteCount(expectedDeletes);
```

Comment:
Nit: seems not necessary to have them in two lines, but just my personal preference.

Reply:
Fixed.